### PR TITLE
Handle unknown asynchronous tooltip response.

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/opengever.core/issues/2825

The tooltip may call the content with a logged out user. This leads
to an unwanted rendering of the whole login page in the tooltip.

So add a response header to make sure the response is a true tooltip response
and redirect the user to the login page with a simple reload.

### CODE REVIEW

**Comment by bierik** [comment]
@phgross We need a way to provide a translated string for rendering the error message.

**Comment by bierik** [comment]
@phgross ping

**Comment by bierik** [comment]
@phgross pingpong

**Comment by bierik** [comment]
As discussed with @phgross we implement a browserview which returns a json containing the translated labels.

**Comment by Rotonen** [comment]
@phgross @bierik why not a `plone.rest` service? That'd be neater here?

**Comment by bierik** [comment]
👍 Of course

**Comment by deiferni** [comment]
@bierik is this thing ready - a bit confused about the PR history here 🤔 ?

**Comment by bierik** [comment]
@phgross updated

File: opengever/base/configure.zcml [hunk 1]
```diff
@@ -110,6 +110,15 @@
       permission="zope2.View"
       />
 
+  <!-- JSON endpoint for javascript translations -->
+  <plone:service
+      method="GET"
+      for="*"
+      factory=".service.Internationalization"
+      name="@i18n"
+      permission="zope.Public"
```
Comment (Rotonen) [inline opengever/base/configure.zcml]: Rather use `zope2.View`? Or the REST API permission?

File: opengever/base/service.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
+from frontend_translations import get_translations
 from opengever.document.interfaces import ICheckinCheckoutManager
-from plone.rest import Service
+from plone.restapi.services import Service
```
Comment (Rotonen) [inline opengever/base/service.py]: What's going on here?

Comment (bierik) [inline opengever/base/service.py]: Right I should put that in a separate commit. Here I was not able to make the `i18n` endpoint working with the `plone.rest` `Service`. So I used the `plone.restapi.services`. This was also @lukasgraf suggestion.

File: opengever/base/configure.zcml (hunk 1 above)
Comment (bierik) [inline opengever/base/configure.zcml]: Anonymous users need to have access to this endpoint as well. What `REST API` permission should I use here?

**Comment by bierik** [comment]
@Rotonen @phgross This PR is just about fixing unknown asynchronous tooltip response.
The implementation for the i18n on client side has been moved to https://github.com/4teamwork/opengever.core/pull/3020
